recipes/metaimageio/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "metaimageio" %}
+{% set version = "1.0.0" %}
+{% set hash = "fa1e0aa1980e10aac3ceb0bfa794b734413f1399c750fd8208dcb2f869ed37df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
```

Comment:
Go ahead and just put the hash in here directly and don't bother with templating it--it's not used in multiple places (ever).